fastlane/lib/fastlane/actions/download_dsyms.rb

```diff
@@ -44,6 +44,8 @@ def self.run(params)
             version = latest_candidate_build.train_version
             build_number = latest_candidate_build.build_version
           end
+        else
+          FastlaneCore::BuildWatcher.wait_for_build_processing_to_be_complete(app_id: app.apple_id, platform: platform, train_version: version, build_version: build_number, poll_interval: 30, strict_build_watch: true)
```

Comment:
Thoughts on adding a new option around this? Similar to the `pilot` one that is called `skip_waiting_for_build_processing`? Even though I think this is 💯 myself, I don't want to have to _force_ users to wait for processing if they don't want to wait for whatever reason 🙃 

I think I'm more concerned with breaking/changing any existing behavior/implementation around how users are currently expecting this work. I'm okay with making this new option default to `true` but just at least give the user an option to opt out 🤔

Reply:
I think that makes sense. I ran into some cases where it would watch indefinitely and never find the build due to other bugs.

Reply:
I guess if you didn't want to introduce breaking changes, the best thing to do might be to have it default to not waiting (old behavior), and introduce a new option that turns your feature on only if the user opts in.  I can see how other people might get confused when this feature hits about why their CI is hanging on some task all of a sudden.